### ISSUE TITLE
paper-input-container error message text 

### ISSUE THREAD

**Issue by uxdesignz** [PolymerElements/paper-input#708]
Actual Result:
Findable Added Content: The form uses dynamically added error messages. Due to the definition of these messages screen reader users may not be notified when the messages appear, may be notified there is an error before completing the form, and may not be able to locate the messages.

Expected Results:
The element with aria-live="assertive" should be present and empty on page load.  The error message text should be injected dynamically into this element as its immediate child element when an error occurs.
